src/acl/class.test.ts

```diff
@@ -35,6 +37,8 @@ import {
   unstable_AclDataset,
   unstable_WithAcl,
 } from "../interfaces";
+import { Quad } from "rdf-js";
```

Comment:
`DataFactory` is already imported, so to be consistent you could use `DataFactory.Quad` instead.

Reply:
No, I think this is needed for the Type `Quad` (i.e. as in `const updatedQuads: Quad[]`), and not for the method `DataFactory.quad()` (i.e. the type `Quad` is not exported from `DataFactory.ts` (at least not the DataFactory exported from N3, which this test uses (which is another whole issue that my previous PR dealt with - i.e. using types from N3 instead of from RDF/JS))).

Reply:
Ah whoops, you're right!